terraform/ec2/win/main.tf

```diff
@@ -221,3 +281,19 @@ data "aws_ami" "latest" {
     values = [var.ami]
   }
 }
+
+#####################################################################
+# Generate template file for EC2 userdata script
+#####################################################################
+data "template_file" "user_data" {
+  template = file("install_and_start_agent.tpl")
+
+  vars = {
+    copy_object = "Copy-S3Object -BucketName ${var.s3_bucket} -Key integration-test/packaging/${var.cwa_github_sha}/amazon-cloudwatch-agent.msi -region ${var.region} -LocalFile $cwAgentInstaller"
```

Comment:
For some reason var replacement was not working for the internal vars of this command and started working when I added the entire command as the var.